packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/upgrade-guides.5/version-2-upgrade-guide.2.tool.md

```diff
@@ -0,0 +1,25 @@
+# Terra Functional Testing - Version 2 Upgrade Guide
+
+The only change in this version is the upgrade from wdio v7. This should cause very little disruption to consuming projects.
+
+## Breaking Changes
+
+### WDIO v6 to v7
+
+A complete list of breaking changes can be found here:
+
+- [WebdriverIO v7 Breaking Changes](https://github.com/webdriverio/webdriverio/blob/main/CHANGELOG.md#boom-breaking-change)
+- [WebdriverIO v6 to v7 Upgrade Guide](https://github.com/webdriverio/webdriverio/blob/main/CHANGELOG.md#boom-breaking-change)
```

Comment:
This link is the same as above. Did you intend to link to https://webdriver.io/docs/v7-migration?